Lib/xml/dom/minidom.py

```diff
@@ -204,6 +219,7 @@ def normalize(self):
         self.childNodes[:] = L
 
     def cloneNode(self, deep):
+        """The Node.cloneNode() method returns a duplicate of the node."""
```

Comment:
Should be "Return a duplicate ...", but needs to explain 'deep'  (I suspect it is a boolean.)  Docstrings are for quick reminders when coding, and 'deep' is not obvious.